front/src/modules/timesStops/TimesStops.tsx

```diff
@@ -34,14 +35,33 @@ const marginRegExValidation = /^\d+(\.\d+)?%$|^\d+(\.\d+)?min\/100km$/;
 
 const formatSuggestedViasToRowVias = (
```

Comment:
Can you move this function in a new file helpers or utils ?

Reply:
Done. As it depended on types and consts, I also moved those to types.ts and consts.ts